visualizer/IdealGraphVisualizer/View/src/main/java/org/graalvm/visualizer/view/impl/GraphTypeModeSelector.java

```diff
@@ -45,7 +46,12 @@
 public class GraphTypeModeSelector implements ModeSelector {
     @Override
     public Mode selectModeForOpen(TopComponent tc, Mode mode) {
-        if (tc.getLookup().lookup(InputGraph.class) == null) {
+        try {
+            if (tc.getLookup().lookup(InputGraph.class) == null) {
```

Comment:
Without this change I couldn't open any file in the editor. I was getting exception like this one

![obrazek](https://github.com/user-attachments/assets/9da24c08-3dd8-4693-8944-7213e5552e1b)